doc/main.xml

```diff
@@ -77,10 +77,11 @@
   <#Include SYSTEM "z-chap16.xml">  <!-- properties and attributes of inverse semigroups -->
```

Comment:
I should specify I have shuffled doc chapters down so that this new "fp semigroups" chapter doesn't appear after the stuff on IO and visualising semigroups!